src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -33,9 +33,21 @@
 public class Datastore {
 
   private DatastoreService datastore;
+  private List<Laptop> laptopList; 
 
+  //in future, change to generic type to go more than laptops 
   public Datastore() {
     datastore = DatastoreServiceFactory.getDatastoreService();
+    laptopList= new ArrayList<>();
+	laptopList.add(new Laptop("Dell","Black","Windows",15,800.00));
```

Comment:
nit: You might want to fix the indentation in this file. The Google style guide says to use spaces instead of tabs: https://google.github.io/styleguide/javaguide.html